docs/content/documentation/systems/Materials/heat_conduction/SemiconductorLinearConductivity.md

```diff
@@ -0,0 +1,25 @@
+<!-- MOOSE Documentation Stub: Remove this when content is added. -->
+
+# SemiconductorLinearConductivity
+!syntax description /Materials/SemiconductorLinearConductivity
+
+For Linear relationship in Steinhart Hart equation, coefficient C = 0 \\
```

Comment:
Use this syntax for web links:
`[text for your hyperlink](https://en.wikipedia.org/wiki/Electrical_resistivity_and_conductivity)`